strapi/strapi
# [Custom Fields] Inject inputs from plugins

### What does it do?

- Extends custom field create attribute form by allowing plugins to inject inputs
- Creates a new section for all injected inputs called 'Extended settings'

### How to test it?

- Go to any content type where i18n is enabled
- Click the create another field button
- Select the custom tab, choose color, and go to the advanced tab. You should see the `Extended settings` section with a checkbox for "Enable localization for this field"
